### PR TITLE
Feat button functionality

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have created a Home page and Video Listing pages with the required functions. It is not responsive yet but will make it responsive as soon as possible.

### CODE REVIEW

File: src/assests/styles/video-listing.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.videos{
+    display:flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+}
+
+.video{
+    padding:0.5rem;
```
Comment (t007rushi) [inline src/assests/styles/video-listing.css]: make it px below 10px

File: src/backend/utils/useAxios.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+function useAxios(apipath) {
```
Comment (t007rushi) [inline src/backend/utils/useAxios.js:4]: nice that you used hooks for axios

File: src/components/Category/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <div className="video-category">
+      <p className="category-heading">Categories</p>
+      <div className="category-container">
+        {loading ? (
+          <h4>Loading...</h4>
+        ) : (
+          categories.map((item) => {
+            return (
+              <div className="video-cat">
```
Comment (t007rushi) [inline src/components/Category/Category.jsx:17]: ```key ``` ?

**Review by t007rushi — APPROVED** [review]
Lok for suggestions else good to go with some minor refactoration

File: src/pages/VideoListing/VideoListing.jsx [hunk 4]
```diff
@@ -1,26 +1,25 @@
...
-              videoTitle={item.title}
-              views={item.views}
-              creator={item.creator}
-              videoUrl={item.videoUrl}
-              videoImage={item.videoImage}
-            />
-          );
-        })}
-      </div>
+      {videos.map((item) => {
```
Comment (sushil-mod) [inline src/pages/VideoListing/VideoListing.jsx:11]: this is right . just a suggestion you can also destructure directly  in map ({ _id, title,views ,creator ,videoUrl.... })

**Review by sushil-mod — APPROVED** [review]
just a suggestion . I like your making short PR  and also the hook for axios .good work . rest code is fine

File: src/pages/VideoListing/VideoListing.jsx (hunk 4 above)
Comment (Kothalkarpriya) [reply src/pages/VideoListing/VideoListing.jsx:11]: Ok I will try it

File: src/components/Category/Category.jsx (hunk 3 above)
Comment (Kothalkarpriya) [reply src/components/Category/Category.jsx:17]: I will include it

File: src/assests/styles/video-listing.css (hunk 1 above)
Comment (Kothalkarpriya) [reply src/assests/styles/video-listing.css]: I changed rem to px for this one

**Pull request merged**